frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useEffect } from 'react';
+import { API } from '../../modules/apis';
+import { Button } from '../shared/button';
+import Logo from './Logo.png';
+import Icon from './Icon.svg';
+export const Navbar = () => {
+  // Fetching API from the backend
+  const [links, setLinks] = useState([]);
+  useEffect(() => {
+    API('/navigation').then(({ data: navigation }) => {
+      setLinks(navigation);
+    });
+  }, []);
+  return (
+    <nav className="flex  items-center justify-between w-5/6  bg-white shadow-lg p-3 my-10 rounded-full m-auto">
+      <div className="mx-12">
+        <img src={Logo} alt="" className="w-16" />
+      </div>
+      <ul className="flex flex-row items-center justify-between ml-56 text-xl">
+        {/* Looping Menu items */}
+        {links.map((link, index) => {
+          return (
+            <a
+              href="/"
+              key={index}
```

Comment:
please use different key than the index when it is possible, why?[read more](https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c)